.circleci/config.yml.in

```diff
@@ -30,6 +30,18 @@ commands:
 #             git fetch --force origin ${CIRCLE_BRANCH}/merge:merged/${CIRCLE_BRANCH}
 #             git checkout "merged/$CIRCLE_BRANCH"
 #           fi
+  designate_upload_channel:
+    description: "inserts the correct upload channel into ${BASH_ENV}"
+    steps:
+      - run:
+          name: adding UPLOAD_CHANNEL to BASH_ENV
+          command: |
+            our_upload_channel=nightly
+            # On tags upload to test instead
+            if [[ -n "${CIRCLE_TAG}" ]]; then
```

Comment:
How does the `CIRCLE_TAG` works? Will we need to first make a release on GitHub (and commit the PR on the tag)  so that the job running on the branch gets activated?
What if we need to run fixes on top of the tag, we will need to create new tags with the fixes?

Reply:
Yes it works exactly that way.

With the upstream `pytorch/pytorch` repository we've piloted this for the `1.5.0` release with pretty good success so I'm hoping to bring that same workflow to all of the downstream repositories.